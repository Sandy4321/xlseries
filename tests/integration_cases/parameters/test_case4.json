{"alignment": "vertical",
 "blank_rows": "False",
 "composed_headers": "True",
 "data_starts": [46, 46, 46, 46, 93, 93, 93, 93],
 "data_ends": [5, 5, 5, 5, 52, 52, 52, 52],
 "frequency": "Q",
 "headers_coord": ["B4", "C4", "D4", "E4", "B51", "C51", "D51", "E51"],
 "continuity": "True",
 "missings": "True",
 "missing_value": "â€¦",
 "multifrequency": "False",
 "progressive_aggregation": "False",
 "series_names": "None",
 "time_composed": "True",
 "time_alignment": 0,
 "time_multicolumn": "False",
 "time_format": "str",
 "time_header": "True",
 "time_header_coord": ["A4", "A4", "A4", "A4", "A51", "A51", "A51", "A51"],
 "time_header_next_to_data": "True"}
