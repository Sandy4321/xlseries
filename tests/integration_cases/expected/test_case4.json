{"Mercado de trabajo - EPH -Evolucion de la situacion ocupacional 1 -Tasas (31 aglomerados relevados) -Actividad":{"1057017600000":46.2783093088,"1064966400000":46.3461303324,"1072915200000":46.1420192544,"1080777600000":46.7270253824,"1088640000000":46.685556551,"1096588800000":46.4927087949,"1104537600000":46.0867504272,"1112313600000":45.995773081,"1120176000000":46.7030759502,"1128124800000":46.4669732461,"1136073600000":46.4406120743,"1143849600000":46.9780503797,"1151712000000":46.7141435716,"1159660800000":46.4324974576,"1167609600000":46.5736943111,"1175385600000":46.3679664236,"1183248000000":null,"1191196800000":45.6288513565,"1199145600000":45.8283941844,"1207008000000":45.8328585772,"1214870400000":45.5462612848,"1222819200000":46.2082910337,"1230768000000":46.5159512304,"1238544000000":46.319213087,"1246406400000":46.0625061749,"1254355200000":46.2689503342,"1262304000000":46.0,"1270080000000":46.1,"1277942400000":45.9,"1285891200000":45.8,"1293840000000":45.8,"1301616000000":46.6,"1309478400000":46.73622635,"1317427200000":46.0807320739,"1325376000000":45.5,"1333238400000":46.2,"1341100800000":46.9,"1349049600000":46.3,"1356998400000":45.8,"1364774400000":46.4,"1372636800000":46.1,"1380585600000":45.6},"Mercado de trabajo - EPH -Evolucion de la situacion ocupacional 1 -Tasas (31 aglomerados relevados) -Empleo":{"1057017600000":38.8057662116,"1064966400000":39.6504275121,"1072915200000":39.5543560669,"1080777600000":39.8798909272,"1088640000000":40.5551176954,"1096588800000":40.9034114686,"1104537600000":40.1330505716,"1112313600000":40.4926020334,"1120176000000":41.5088685681,"1128124800000":41.8037825602,"1136073600000":41.1889514588,"1143849600000":42.127609034,"1151712000000":41.9985384978,"1159660800000":42.4279882927,"1167609600000":42.0610127967,"1175385600000":42.5250896634,"1183248000000":null,"1191196800000":42.1964361315,"1199145600000":42.0197926591,"1207008000000":42.187285552,"1214870400000":42.0050770148,"1222819200000":42.8449640696,"1230768000000":42.6267307673,"1238544000000":42.2712941321,"1246406400000":41.8765633258,"1254355200000":42.3952313934,"1262304000000":42.2,"1270080000000":42.5,"1277942400000":42.5,"1285891200000":42.4,"1293840000000":42.4,"1301616000000":43.2,"1309478400000":43.3583358268,"1317427200000":42.9726416138,"1325376000000":42.3,"1333238400000":42.8,"1341100800000":43.3,"1349049600000":43.1,"1356998400000":42.2,"1364774400000":43.1,"1372636800000":42.9,"1380585600000":42.7},"Mercado de trabajo - EPH -Evolucion de la situacion ocupacional 1 -Tasas (31 aglomerados relevados) -Desocupacion":{"1057017600000":16.146966492,"1064966400000":14.447166942,"1072915200000":14.2769286953,"1080777600000":14.6534781513,"1088640000000":13.1313393445,"1096588800000":12.021879282,"1104537600000":12.9184631166,"1112313600000":11.9645147347,"1120176000000":11.1217671993,"1128124800000":10.0354948046,"1136073600000":11.3083363483,"1143849600000":10.3249098388,"1151712000000":10.0945981522,"1159660800000":8.624367381,"1167609600000":9.6893355382,"1175385600000":8.2877836934,"1183248000000":null,"1191196800000":7.5224668667,"1199145600000":8.3105716294,"1207008000000":7.9540599003,"1214870400000":7.7749175676,"1222819200000":7.2786222751,"1230768000000":8.3610468241,"1238544000000":8.7391790256,"1246406400000":9.0875273552,"1254355200000":8.3721781297,"1262304000000":8.3,"1270080000000":7.9,"1277942400000":7.5,"1285891200000":7.3,"1293840000000":7.4,"1301616000000":7.3,"1309478400000":7.2275636845,"1317427200000":6.7448808214,"1325376000000":7.1,"1333238400000":7.2,"1341100800000":7.6,"1349049600000":6.9,"1356998400000":7.9,"1364774400000":7.2,"1372636800000":6.8,"1380585600000":6.4},"Mercado de trabajo - EPH -Evolucion de la situacion ocupacional 1 -Tasas (31 aglomerados relevados) -Subocupacion":{"1057017600000":16.5357119166,"1064966400000":16.1702259642,"1072915200000":15.4399908137,"1080777600000":15.1055055251,"1088640000000":15.031435011,"1096588800000":14.119129675,"1104537600000":12.4638875341,"1112313600000":12.7523970629,"1120176000000":12.9693777258,"1128124800000":11.8227642344,"1136073600000":10.9751777196,"1143849600000":12.0514288258,"1151712000000":10.8532474874,"1159660800000":10.7414646247,"1167609600000":9.2714701793,"1175385600000":10.001127029,"1183248000000":null,"1191196800000":9.109353219,"1199145600000":8.2652787881,"1207008000000":8.6654362189,"1214870400000":9.3847496526,"1222819200000":9.1256417509,"1230768000000":9.040920379,"1238544000000":10.5132619116,"1246406400000":10.6,"1254355200000":10.2949496961,"1262304000000":9.2,"1270080000000":9.9,"1277942400000":8.8,"1285891200000":8.4,"1293840000000":8.2,"1301616000000":8.4,"1309478400000":8.7981275073,"1317427200000":8.5,"1325376000000":7.4,"1333238400000":9.4,"1341100800000":8.9,"1349049600000":9.0,"1356998400000":8.0,"1364774400000":9.7,"1372636800000":8.7,"1380585600000":7.8},"Poblacion (Total urbano 3 - en miles de personas) -Poblacion total":{"1057017600000":33845.095,"1064966400000":33944.84,"1072915200000":34044.586,"1080777600000":34141.078,"1088640000000":34241.779,"1096588800000":34343.478,"1104537600000":34445.178,"1112313600000":34543.561,"1120176000000":34646.509,"1128124800000":34750.762,"1136073600000":34855.014,"1143849600000":34955.867,"1151712000000":35060.817,"1159660800000":35166.495,"1167609600000":35270.0215548191,"1175385600000":35373.419633676,"1183248000000":35480.4171996887,"1191196800000":35587.5388112462,"1199145600000":35690.268730893,"1207008000000":35796.2887230446,"1214870400000":35905.380361063,"1222819200000":36013.9644361902,"1230768000000":36115.8245157212,"1238544000000":36224.5450462672,"1246406400000":36335.7782199769,"1254355200000":36445.8438542551,"1262304000000":36546.7588305473,"1270080000000":36658.2603509701,"1277942400000":36771.6834997754,"1285891200000":36883.2500355107,"1293840000000":36983.1425432161,"1301616000000":37097.5073677534,"1309478400000":37213.1699251225,"1317427200000":37326.2569545419,"1325376000000":37425.047481209,"1333238400000":37542.3598238881,"1341100800000":37660.3122360233,"1349049600000":37774.9396045132,"1356998400000":37872.5464448608,"1364774400000":37992.8924572083,"1372636800000":38113.1862020452,"1380585600000":38229.3740114317},"Poblacion (Total urbano 3 - en miles de personas) -PEA":{"1057017600000":15326.9704005674,"1064966400000":15453.4229133733,"1072915200000":15446.7239543016,"1080777600000":15626.6097102414,"1088640000000":15633.4159177753,"1096588800000":15608.7976585411,"1104537600000":15541.2949761025,"1112313600000":15581.063513978,"1120176000000":15832.6448662139,"1128124800000":15855.1861160499,"1136073600000":15823.2819990658,"1143849600000":16054.477896583,"1151712000000":16067.0288064282,"1159660800000":16031.3827282126,"1167609600000":16098.7700693755,"1175385600000":16060.3579475433,"1183248000000":null,"1191196800000":15981.4861787319,"1199145600000":16107.373395699,"1207008000000":16161.9854475531,"1214870400000":16099.5571570608,"1222819200000":16347.5803269354,"1230768000000":16471.271442746,"1238544000000":16464.2410297382,"1246406400000":16428.7395473493,"1254355200000":16531.2640886481,"1262304000000":16535.9227189705,"1270080000000":16567.0890126887,"1277942400000":16525.6351514031,"1285891200000":16536.2192726126,"1293840000000":16664.4837134822,"1301616000000":16977.9383154889,"1309478400000":17075.7280720393,"1317427200000":16884.26299176,"1325376000000":16816.7041818887,"1333238400000":17122.0930833045,"1341100800000":17310.05658562,"1349049600000":17151.047912205,"1356998400000":17120.8142112765,"1364774400000":17366.4244427167,"1372636800000":17305.7202992905,"1380585600000":17238.167100477},"Poblacion (Total urbano 3 - en miles de personas) -Ocupados":{"1057017600000":12933.1981646246,"1064966400000":13288.8706593676,"1072915200000":13258.3803058694,"1080777600000":13408.0837288692,"1088640000000":13640.9184455872,"1096588800000":13796.5593118358,"1104537600000":13627.487558015,"1112313600000":13801.4975326481,"1120176000000":14140.9242419484,"1128124800000":14324.8796693687,"1136073600000":14099.9093980366,"1143849600000":14453.6731778458,"1151712000000":14510.5867461054,"1159660800000":14714.8565792478,"1167609600000":14614.3883744786,"1175385600000":14754.8731220571,"1183248000000":null,"1191196800000":14795.473726041,"1199145600000":14807.3854070593,"1207008000000":14903.3055147761,"1214870400000":14887.5345940183,"1222819200000":15180.64442987,"1230768000000":15104.5933292311,"1238544000000":15030.7169123782,"1246406400000":14982.5664024071,"1254355200000":15191.7149219592,"1262304000000":15178.6937672844,"1270080000000":15286.7545673447,"1277942400000":15344.4056033342,"1285891200000":15377.5875328346,"1293840000000":15461.4909499004,"1301616000000":15761.9607656261,"1309478400000":15871.5892602857,"1317427200000":15805.0519850926,"1325376000000":15639.9381625764,"1333238400000":15913.4214470867,"1341100800000":16048.9073484765,"1349049600000":16032.8218979676,"1356998400000":15823.3457567556,"1364774400000":16133.6758762664,"1372636800000":16155.2659884433,"1380585600000":16141.0450759522},"Poblacion (Total urbano 3 - en miles de personas) -Desocupados":{"1057017600000":2393.7722359428,"1064966400000":2164.5522540058,"1072915200000":2188.3436484322,"1080777600000":2218.5259813721,"1088640000000":1992.497472188,"1096588800000":1812.2383467054,"1104537600000":1913.8074180875,"1112313600000":1779.5659813299,"1120176000000":1691.7206242655,"1128124800000":1530.3064466811,"1136073600000":1723.3726010291,"1143849600000":1600.8047187372,"1151712000000":1556.4420603228,"1159660800000":1316.5261489647,"1167609600000":1484.3816948969,"1175385600000":1305.4848254861,"1183248000000":null,"1191196800000":1186.0124526908,"1199145600000":1299.9879886397,"1207008000000":1258.679932777,"1214870400000":1212.0225630425,"1222819200000":1166.9358970654,"1230768000000":1366.6781135148,"1238544000000":1433.52411736,"1246406400000":1446.1731449422,"1254355200000":1339.5491666888,"1262304000000":1357.2289516861,"1270080000000":1280.334445344,"1277942400000":1181.2295480688,"1285891200000":1158.631739778,"1293840000000":1202.9927635817,"1301616000000":1215.9775498629,"1309478400000":1204.1388117537,"1317427200000":1079.2110066674,"1325376000000":1176.7660193123,"1333238400000":1208.6716362178,"1341100800000":1261.1492371434,"1349049600000":1118.2260142374,"1356998400000":1297.4684545209,"1364774400000":1232.7485664503,"1372636800000":1150.4543108472,"1380585600000":1097.1220245248}}